db/migrate/20230831075724_add_role_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddRoleToUsers < ActiveRecord::Migration[7.0]
+  def change
+    add_column :users, :role, :string
+  end
+end
```

Comment:
- [ ] Great job so far 👏🏻 However, as I've discussed on call, you do have some conflict in your code that you need to address and I can't even run the Migration as well 👇🏻 
![image](https://github.com/ANTOINE1128/Blog-App1/assets/79058364/f96abfc4-ed37-402c-85c3-7797b4a53381)

- [ ] Also the **POST :** `/users/:user_id/posts/:id/comments`. seem not to be functioning as expected. Please fx all of the issues and submit for the re-review!! 🧑‍🔧